src/style.css

```diff
@@ -0,0 +1,99 @@
+/* import stars.css and styling the card container */
+@import "./stars.css";
+
+.card {
+  font-family: "Roboto", sans-serif;
+  width: 198px;
+  height: 406px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  --primary-color: #060b35;
+  --secondary-color: #616070;
+  --accent-color: #00acdc;
+}
+
+/* image and title and product code styling */
+.image {
+  margin: 32px 19px 0;
+  width: 160px;
+  height: 134px;
+}
+
+.title {
+  margin: 36px 16px 0;
+  font-size: 12px;
+  line-height: 18px;
+  color: var(--primary-color)
+  font-weight: 500;
+}
+
+.code {
+  margin: 4px 16px 0;
+  font-size: 10px;
+  color: var(--secondary-color);
+  height: 14px;
+  line-height: 14px;
+}
```

Comment:
Make all these and below elements of `card` block.
These are not blocks by themselves, you wouldn't reuse them separately.